### ISSUE TITLE
can't use tf.nn.moments in tflite

### ISSUE THREAD

**Issue by mmmmayi** [tensorflow/tensorflow#32112]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 18.04
- TensorFlow installed from (source or binary): 
- TensorFlow version (or github SHA if from source):Tensorflow nightly

Actually, error doesn't happened when I use the function of tf.nn.moments in code:
`mean, var = tf.nn.moments(x, [2, 3, 4], keep_dims=True)`
but when i use the tflite model, `aborted (core dumped)` happened, then I realize tflite doesn't support it, but it really important for me. So could you help me to add the function into tflite or just recommend a similar function to me? Thanks a lot

**Comment by oanush** [comment]
@mmmmayi,
Can you share a simple and standalone code to reproduce the issue?
Also mentioned TF version being used.Thanks!

**Comment by mmmmayi** [comment]
> @mmmmayi,
> Can you share a simple and standalone code to reproduce the issue?
> Also mentioned TF version being used.Thanks!

I use it in the function to calculate group normalization, same as nn.GroupNorm() in PyTorch:
```
def GroupNorm(x, G, eps=1e-05):
    # x: input features with shape [N,H,W,C]
    # G: number of groups for GN
  N, H, W, C = x.shape
  gamma = tf.ones([1, 1, 1, C])
  beta = tf.zeros([1, 1, 1, C])
  x = tf.reshape(x, [N, G, H, W, C // G])
  mean, var = tf.nn.moments(x, [2, 3, 4], keep_dims=True)
  x = (x-0) / tf.sqrt(1 + eps)
  x = tf.reshape(x, [N, H, W, C])
  return x * gamma + beta
```
TF version being used is: 1.15.0-dev20190821

**Comment by ymodak** [comment]
The recommended approach is to [convert the model](https://www.tensorflow.org/lite/guide/ops_select#converting_the_model) with TFLITE_BUILTINS, then with both TFLITE_BUILTINS,SELECT_TF_OPS, and finally with only SELECT_TF_OPS. Using both options (i.e. TFLITE_BUILTINS,SELECT_TF_OPS) creates models with TensorFlow Lite ops where possible.
Further if problem still persists, you have to add custom implementation to add this op.
See https://www.tensorflow.org/lite/guide/ops_custom
